wpilibc/src/main/native/include/frc/simulation/SimBattery.h

```diff
@@ -0,0 +1,59 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <units/current.h>
+#include <units/impedance.h>
+#include <units/voltage.h>
+
+namespace frc {
+
+namespace sim {
+
+class SimBattery {
+ public:
+  /**
+   * Calculate the loaded battery voltage. Use this with
+   * {@link RoboRioSim#setVInVoltage(double)} to set the simulated battery
+   * voltage, which can then be retrieved with the {@link
+   * RobotController#getBatteryVoltage()} method.
+   *
+   * @param nominalVoltage The nominal battery voltage. Usually 12v.
+   * @param resistance The forward resistance of the battery. Most batteries
+   * are at or below 20 milliohms.
+   * @param currents       The currents drawn from the battery.
+   * @return The battery's voltage under load.
+   */
+  static units::volt_t Calculate(
+      units::volt_t nominalVoltage, units::ohm_t resistance,
+      std::initializer_list<units::ampere_t> currents) {
+    auto retval = nominalVoltage;
+    for (const units::ampere_t current : currents) {
+      retval -= current * resistance;
+    }
+    return retval;
```

Comment:
Add `#include <numeric>` for `std::accumulate()` and do this instead:
```suggestion
    return nominalVoltage - std::accumulate(currents.begin(), currents.end(), 0_A) * resistance;
```